test/katex-spec.js

```diff
@@ -1755,8 +1755,8 @@ describe("An accent parser", function() {
         expect(parse.type).toEqual("supsub");
     });
 
-    it("should not parse expanding accents", function() {
-        expect("\\widehat{x}").toNotParse();
+    it("should parse expanding accents", function() {
+        expect("\\widehat{x}").toParse();
```

Comment:
A `.toBuild()` test should exercise different code paths for shifted accents.  It's less important once we have the screenshot tests.